libs/core/src/components/pds-combobox/pds-combobox.tsx

```diff
@@ -99,6 +99,35 @@ export class PdsCombobox implements BasePdsProps {
    */
   @Prop() triggerVariant: 'secondary' | 'primary' | 'accent' | 'destructive' = 'secondary';
 
+  /**
+   * The sentiment for the chip trigger. Matches Pine chip sentiments.
+   * @default 'neutral'
+   */
+  @Prop() chipSentiment: 'accent' | 'brand' | 'danger' | 'info' | 'neutral' | 'success' | 'warning' = 'neutral';
```

Comment:
Maybe worth adding these to the `libs/core/src/utils/types.ts file` file so we can share them between the chip component and this one. Then we don't need to maintain in two locations.